kategory-docs/docs/docs/typeclasses/eq/README.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: docs
+title: Eq
+permalink: /docs/typeclasses/eq/
+---
+
+## Eq
+
+The `Eq` typeclass abstracts the ability to compare two instances of any object.
+Depending on your needs this comparison can be structural -the content of the object-, referential -the memory address of the object-, based on an identity -like an Id fields-, or any combination of the above.
+
+It can be considered the typeclass equivatent of Java's `Object#equals`.
+
+### Main Combinators
+
+#### eqv
+
+Compares two instances of F and returns true if they're considered equal for this instance.
+It is the opposite comparison of `neqv`.
+
+`fun eqv(a: F, b: F): Boolean`
+
+```kotlin:ank
+Option.pure(1) // Option(1)
```

Comment:
remove `// Option(1)` as `kotlink:ank` already adds that unless you use `kotlin:ank:silent`